pkg/cmd/stack/translate_test.go

```diff
@@ -120,8 +120,9 @@ func Test_translateDeployment(t *testing.T) {
 	}
 	assert.Equal(t, result.Labels, labels)
 	annotations := map[string]string{
-		"annotation1": "value1",
-		"annotation2": "value2",
+		"annotation1":           "value1",
+		"annotation2":           "value2",
+		"dev.okteto.com/sample": "true",
```

Comment:
This (and the rest of changes in this file) is needed because, as the tests run within `okteto/okteto` repository, it is being detected and added the annotation because it is considered an Okteto sample. I tried to change it, but there is no easy way to do it as it uses current working directory internally